tests/unit/config/argv_test.c

```diff
@@ -37,32 +37,22 @@ MTF_DEFINE_UTEST(argv_test, deserialize_to_params_invalid_param)
 MTF_DEFINE_UTEST(argv_test, deserialize_to_params_invalid_value)
 {
     merr_t             err;
-    const char * const paramv[] = { "fanout=0" };
+    const char * const paramv[] = { "prefix.length=999999999" };
     struct kvs_cparams params;
 
     err = argv_deserialize_to_kvs_cparams(1, paramv, &params);
     ASSERT_NE(0, err);
 }
 
-MTF_DEFINE_UTEST(argv_test, deserialize_to_params_invalid_relation_validation)
-{
-    merr_t             err;
-    const char * const paramv[] = { "cn_node_size_hi=49" };
-    struct kvs_rparams params;
-
-    err = argv_deserialize_to_kvs_rparams(NELEM(paramv), paramv, &params);
-    ASSERT_NE(0, err);
-}
-
 MTF_DEFINE_UTEST(argv_test, deserialize_to_params)
 {
     merr_t             err;
-    const char * const paramv[] = { "fanout=8", "prefix.length=7" };
+    const char * const paramv[] = { "suffix.length=5", "prefix.length=7" };
```

Comment:
Since suffix.length is going away in a future PR, might just be easier to remove it, and leave the array with a length of 1.

Reply:
Roger.

Reply:
Done.